src/msg/async/Event.cc

```diff
@@ -16,6 +16,7 @@
 
 #include "include/compat.h"
 #include "common/errno.h"
+#include <cerrno>
```

Comment:
Is this actually needed?  There is already a few instances of `EAGAIN` and `ERANGE` use in this file -- how is `ENOENT` different?

Reply:
it depends on what "needed" means. i think, we should not rely on other headers to bring in the stuff we need, unless they are designed to provide these symbols / declarations.

Reply:
> unless they are designed to provide these symbols / declarations.

In this case, I think this can be said about `include/compat.h` which is supposed to provide a consistent set of constants to work with, including error codes.

Reply:
i am not quite sure that `include/compat.h` is better than `<cerrno>` when it comes to the errno defined by C++ standard. see https://eel.is/c++draft/errno . but if you insist, i would change. but still, i think the standard is well established in this case.

Reply:
Since you intended to add it, I don't insist at all.  It's just when I saw the include, it caught me by surprise with a thought along the lines of "Is there really not single error code already used in this file?", so I commented.

Reply:
wanted to take this opportunity to improve the quality of the code instead of letting it go and creating a "minimal" fix.